packages/ui-scripts/lib/icons/build-icons.js

```diff
@@ -42,9 +43,9 @@ export default {
   },
   handler: async (argv) => {
     const configFile = path.join(process.cwd(), argv.config)
-    const config = await import(configFile)
+    const configFileURL = pathToFileURL(configFile)
+    const config = await import(configFileURL)
```

Comment:
`import()` for files must begin with `file://`, this is added by this built in helper